packages/wonder-blocks-modal/components/modal-launcher.test.js

```diff
@@ -206,4 +209,48 @@ describe("ModalLauncher", () => {
         wrapper.simulate("click");
         expect(onClose).not.toHaveBeenCalled();
     });
+
+    test("return focus to the last element focused outside the modal", async () => {
+        // Arrange
+        let savedCloseModal = () => {
+            throw new Error(`closeModal wasn't saved`);
+        };
+
+        const wrapper = mount(
+            <ModalLauncher
+                modal={({closeModal}) => {
+                    savedCloseModal = closeModal;
+                    return exampleModal;
+                }}
+            >
+                {({openModal}) => (
+                    <button onClick={openModal} data-last-focused-button />
+                )}
+            </ModalLauncher>,
+        );
+
+        // Act
+        const lastButton = wrapper
+            .find("[data-last-focused-button]")
+            .getDOMNode();
+        // force focus
+        lastButton.focus();
+
+        // Launch the modal.
+        wrapper.find("button").simulate("click");
+
+        // wait for styles to be applied
+        await sleep();
+
+        // focus has been moved inside the modal
+        expect(document.activeElement).not.toBe(lastButton);
+
+        // Close the modal.
+        savedCloseModal();
+        wrapper.update();
```

Comment:
This is really the `Act` part of the tests based on the what the test title is and everything before this is `Arrange`.